server/player.lua

```diff
@@ -904,7 +904,7 @@ function CreatePlayer(playerData, Offline)
     AddEventHandler('qbx_core:server:onJobUpdate', function(jobName, job)
         if self.PlayerData.job.name ~= jobName then return end
         if not job then
-            self.Functions.setJob('unemployed', 0)
+            SetJob(self.PlayerData.source, 'unemployed', 0)
```

Comment:
I wonder if this line should be deleted altogether?

Reply:
Do you remember why you added it initially?

Reply:
The gang version below doesn't appear to do the same set. I think these were from before qbx_management used exports for the multijob?

Reply:
I'm not sure why this line is here. If the event is triggered, I assume it is because the player's job is already unemployed, so no need to set it again.

Reply:
I've changed it to now mirror onGangUpdate.

Reply:
To answer the purpose we listen to this event for the RemoveJob/AddJob exports so that other folks could integrate this into their scripts via the netevent.